pkg/languagedetection/detector.go

```diff
@@ -39,6 +39,14 @@ var exactMatches = map[string]languageFromCLI{
 	"node": {name: languagemodels.Node},
 
 	"dotnet": {name: languagemodels.Dotnet},
+
+	"rails":   {name: languagemodels.Ruby},
+	"ruby":    {name: languagemodels.Ruby},
+	"rubyw":   {name: languagemodels.Ruby},
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏
Shouldn't we just add `ruby` to `knownPrefixes`?
Same for `puma`

Reply:
Hey sorry for the late reply! The reason I added them here is because I didn't see any scenarios where a prefix is used. I'm not the most familiar with ruby, so this is just based what I've seen in the wild. Let me know if I'm missing something here, or if this assumption is incorrect!